src/iop/toneequal.c

```diff
@@ -126,35 +128,60 @@
 
 DT_MODULE_INTROSPECTION(2, dt_iop_toneequalizer_params_t)
```

Comment:
You need to update the version here to support old edits - legacy_params() only gets called if the stored version is less than the version given here (and new edits would get confused with old ones without the version bump).